### PR TITLE
Enable deleting template division

### PR DESCRIPTION
**Pull request by jenachoi**

### CODE REVIEW

**Comment by jenachoi** [comment]
destroy_all method doesn't seem to delete all exam templates yet. I'll be working on it soon

File: app/views/exam_templates/_form.html.erb [hunk 1]
```diff
@@ -3,6 +3,12 @@
     <%= f.error_messages%>
     <div class='template-pane'>
       <h4>
+        <%= link_to t('exam_templates.delete.all_general'), action: 'destroy_all' %>
```
Comment (david-yz-liu) [inline app/views/exam_templates/_form.html.erb]: This is not necessary; you can remove this button (and route).

File: app/views/exam_templates/_form.html.erb [hunk 2]
```diff
@@ -3,6 +3,12 @@
...
+      </h4>
+      <h4>
+        <%= link_to t('exam_templates.delete.general'), action: 'destroy', id: exam_template.id %>
```
Comment (david-yz-liu) [inline app/views/exam_templates/_form.html.erb]: This is not the normal Rails approach. See http://guides.rubyonrails.org/form_helpers.html#removing-objects.

Comment (jenachoi) [inline app/views/exam_templates/_form.html.erb]: I think the link is about deleting associated objects, which might be useful to delete template divisions. But for deleting exam template objects, aren't we supposed to write our own method in the controller?

File: app/views/exam_templates/_form.html.erb [hunk 3]
```diff
@@ -22,6 +22,7 @@
       <p><%= raw t('assignment.required_fields') %> <span class='required_field'>*</span></p>
       <%= f.fields_for :template_divisions do |form| %>
         <p>
+          <%= form.check_box :_destroy%>
```
Comment (david-yz-liu) [inline app/views/exam_templates/_form.html.erb]: There should be a space before the `%>`. Also, this checkbox should have a label.

**Pull request merged**